.sandbox/provision/lifecycle-asg/on-create.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+ASG_NAME=test-windows-provision
+AVAILABILITY_ZONE=us-east-2a
+MAX_RETRIES=3
+
+source ./common.sh
+
+# adjust the terminal output settings 
+redirect_output on-create.log
```

Comment:
Better to avoid this in a reference solution. The user should use Crafting the logging system.

Reply:
For life cycle based handlers, our logging system captures all the stdout and save it to the state file. I don't think that is what we want in this solution. To redirect the stdout and stderr can make the state file much cleaner.

Reply:
Now I understand. Please add comment explaining why we do this - this is reference solution for customers, not internal code. And please redirect all output to stderr, so users can still see the logs from the UI. No need to use fd 3 and 4.